tests/test_ingesters.py

```diff
@@ -877,6 +887,18 @@ def test_normalize_netcdf_attributes_with_nansat(self):
         self.assertEqual(
             normalized_attributes['gcmd_location']['Location_Category'], 'VERTICAL LOCATION')
         self.assertEqual(normalized_attributes['gcmd_location']['Location_Type'], 'SEA SURFACE')
+        self.assertEqual(
+            normalized_attributes['dataset_parameters'],
+            [
+                OrderedDict(
+                    [('standard_name', 'surface_backwards_scattering_coefficient_of_radar_wave'),
+                     ('canonical_units', '1'),
+                        ('grib', ''),
+                        ('amip', ''),
+                        ('description',
+                         'The scattering/absorption/attenuation coefficient is assumed to be an integral over all wavelengths, unless a coordinate of radiation_wavelength is included to specify the wavelength. Scattering of radiation is its deflection from its incident path without loss of energy. Backwards scattering refers to the sum of scattering into all backward angles i.e. scattering_angle exceeding pi/2 radians. A scattering_angle should not be specified with this quantity.')
```

Comment:
This line is too long, if you need to you can have a look at unit tests in metanorm to see how to break it and fit in the characters limit.

Reply:
it is a description only. DOes it matter to be in limit? I dont think so. It is just a string.

Reply:
It does matter. The character limit applies to the whole code, including strings.

Reply:
Done